modules/era/sql/synth_recipes_era.sql

```diff
@@ -18,453 +18,3476 @@ ALTER TABLE `synth_recipes`
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Volt Gun' AND ID = 22536;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wailing Shell' AND ID = 50010;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Walahra Burner' AND ID = 24519;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wamoura Cloth' AND ID = 34051;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wamoura Silk' AND ID = 34044;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wamoura Silk' AND ID = 34046;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wamoura Silk' AND ID = 34048;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='War Aketon' AND ID = 34033;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='War Beret' AND ID = 44012;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='War Boots' AND ID = 44033;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wardrobe' AND ID = 4548;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Water Lamp' AND ID = 24564;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Water Mitts' AND ID = 33009;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='White Mouton' AND ID = 43026;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='White Mouton' AND ID = 43027;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='White Puppet Turban' AND ID = 33050;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wind Fan' AND ID = 53002;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wind Lamp' AND ID = 24565;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Windurstian Baghnakhs' AND ID = 50504;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wise Braconi' AND ID = 44022;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wise Cap' AND ID = 44050;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wise Gloves' AND ID = 44054;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wise Pigaches' AND ID = 44024;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Witch Nougat' AND ID = 73553;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wivre Gorget' AND ID = 54024;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wivre Hairpin' AND ID = 54021;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wivre Mask' AND ID = 44064;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wivre Ring' AND ID = 54509;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wivre Shield' AND ID = 44002;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wootz Ingot' AND ID = 14597;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wulong Shoes' AND ID = 34014;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Wyvern Helm' AND ID = 54558;
```

Comment:
Allakhazam has 2004 an entry for Wyvern Skull recipe, making this Zilart/Null. 
https://web.archive.org/web/20040423034720/https://ffxi.allakhazam.com/db/item.html?fitem=3642
There's been some fighting back and forth if Wyvern of Guivre Skull can be used, but as far as I can tell it had been settled before ToAU release:
https://web.archive.org/web/20060209000051/http://ffxi.allakhazam.com:80/db/item.html?fitem=3642